docs/htdocs/info/docs/tools/vep/script/vep_download.html

```diff
@@ -116,14 +116,19 @@ <h2 id="new">What's new?</h2>
     <h4>New in version [[SPECIESDEFS::ENSEMBL_VERSION]] <i>([[SPECIESDEFS::ENSEMBL_RELEASE_DATE]])</i></h4>
 
     <ul>
-      <li>New options:
+      <li>VEP options:
         <ul>
-          <li><a href="vep_options.html#opt_var_synonyms">--var_synonyms</a>: Report known synonyms for colocated variants. Must be used with <a href="vep_options.html#opt_cache">--cache</a>.</li>
+          <li><a href="vep_options.html#opt_uniprot">--uniprot</a>: Now we report precise Ensembl translation to UniProt isoform mappings.</li>
```

Comment:
We could also add:
--spdi - https://github.com/Ensembl/ensembl-vep/pull/828
Variant Recoder allele specific changes - https://github.com/Ensembl/ensembl-vep/pull/839

Reply:
vep#839 is for the next release

Reply:
have added `--spdi` option added in the latest commit